python/lsst/ts/salobj/remote.py

```diff
@@ -34,8 +37,15 @@ class Remote:
 
```

Comment:
There is a typo here in 

"If a SAL **conponent** listens to or commands other SAL components"

Reply:
Fixed. I also looked for other instances of the same misspelling and did not find any.